src/group_theory/perm/cycle_type.lean

```diff
@@ -223,6 +224,22 @@ begin
   exact one_lt_two,
 end
 
+lemma subgroup_eq_top_of_mem_swap [decidable_eq α] {H : subgroup (perm α)} {τ : perm α}
+  (h0 : (fintype.card α).prime) (h1 : fintype.card α ∣ fintype.card H) (h2 : τ ∈ H)
```

Comment:
@eric-wieser This is where `fintype.card H` is used.

Reply:
You should add the missing decidable assumption to this lemma